django_girls_tutorial/step2/jinho/djangogirls/blog/api/views.py

```diff
@@ -87,3 +86,25 @@ def update_post_with_put(request, id):
 
     FYI, request.body 활용
     """
+
+
+@require_http_methods(["DELETE"])
+def remove_post_with_delete(request, id):
+    body = json.loads(request.body)
+    try:
+        to_delete_post = Post.objects.get(id=id)
+    except Post.DoesNotExist:
+        return JsonResponse({"message": "post를 찾을 수 없습니다."}, status=NOT_FOUND)
+
+    if body["author"] != to_delete_post.author_id:
+        return JsonResponse({"message": "권한이 없습니다."}, status=UNAUTHORIZED)
+
+    to_delete_post.delete()
+    return JsonResponse(
+        {
+            "post":
+                {
+                    "title": to_delete_post.title,
+                    "text": to_delete_post.text,
+                }
+        }, status=OK)
```

Comment:
200으로 줘도 무방하지만, 대개는 `204 NO_CONTENT`로 주고, data는 따로 보내지 않음

```suggestion
    return JsonResponse(status=204)
```